### PR TITLE
Add Elkfinance on Avax c-chain to dex.trades

### PR DESCRIPTION
**Pull request by blanchemaria6**
### Contribution type
Please check the type of contribution this pull request is for:

- [ ] New spell(s)
- [x] Adding to existing spell lineage
- [ ] Bug fix

### CODE REVIEW

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @blanchemaria6 

[Sample elk_finance_avalanche_c_base_trades]()

File: seeds/_sector/dex/elk_finance_avalanche_c_base_trades_seed.csv [hunk 1]
```diff
@@ -0,0 +1,2 @@
+blockchain,project,version,block_date,tx_hash,evt_index,token_bought_address,token_sold_address,block_number,token_bought_amount_raw,token_sold_amount_raw
+avalanche_c,elk_finance,1,2024-06-14,0x29588b5d8ec3bee204f05abeb218864ab03d50ef89e72f6273317ab4daa677a6,27,0xb31f66aa3c1e785363f0875a1b74e27b85fd66c7,0xeeeeeb57642040be42185f49c52f7e9b38f8eeee,46711000,9858382649191728144,2817240000000000000000
```
Comment (Hosuke) [inline seeds/_sector/dex/elk_finance_avalanche_c_base_trades_seed.csv:2]: Please recheck this seed record.

**Review by Hosuke — CHANGES_REQUESTED** [review]
The seed record is not existed in the query result.

File: seeds/_sector/dex/elk_finance_avalanche_c_base_trades_seed.csv (hunk 1 above)
Comment (blanchemaria6) [reply seeds/_sector/dex/elk_finance_avalanche_c_base_trades_seed.csv]: I just fixed @Hosuke. Thank you for pointing out

**Review by Hosuke — APPROVED** [review]

File: dex/seeds/trades/elk_finance_avalanche_c_base_trades_seed.csv [hunk 2]
```diff
@@ -0,0 +1,2 @@
+blockchain,project,version,block_date,tx_hash,evt_index,token_bought_address,token_sold_address,block_number,token_bought_amount_raw,token_sold_amount_raw
+avalanche_c,elk_finance,1,2024-06-18,0xf39bfaa3b700a97c9a635b39619932a0e38610922e1a1fba90adf5c5c51ed649,58,0xb31f66aa3c1e785363f0875a1b74e27b85fd66c7,0xeeeeeb57642040be42185f49c52f7e9b38f8eeee,46852217,1352121299455297547805,4945531145165399399
```
Comment (Hosuke) [inline dex/seeds/trades/elk_finance_avalanche_c_base_trades_seed.csv:2]: This two addresses are still flipped:
```suggestion
avalanche_c,elk_finance,1,2024-06-18,0xf39bfaa3b700a97c9a635b39619932a0e38610922e1a1fba90adf5c5c51ed649,58,0xeeeeeb57642040be42185f49c52f7e9b38f8eeee,0xb31f66aa3c1e785363f0875a1b74e27b85fd66c7,46852217,1352121299455297547805,4945531145165399399
```

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**